core/src/main/java/org/hisp/dhis/android/persistence/dataelement/DataElementDB.kt

```diff
@@ -57,4 +65,50 @@ internal data class DataElementDB(
     val fieldMask: String?,
     val color: String?,
     val icon: String?,
```

Comment:
What do you think about creating a `ObjectWithStyleDB` interface for these two fields? and adding an `applyStyleFields` method in the mapper? Or maybe it is too much duplication of classes in domain and db...